java/runtime/src/main/resources/ray.default.conf

```diff
@@ -118,5 +118,9 @@ ray {
 
   // Below args will be appended as parameters of the `ray start` command.
   // It takes effect only if Ray head is started by a driver.
-  head-args: []
+  head-args: [
+//    "--num-cpus=1",
+//    "--num-gpus=1",
+//    "--memory=1073741824"
```

Comment:
Also add a "labels" case here?

Reply:
Users can add any parameters in head-args.
But the `--labels` parameter is not commonly used, so i don't add `--labels` case here.